docs/rfc/README.md

```diff
@@ -48,5 +48,11 @@ sections.
 - [RFC-008: Don't Panic](./rfc-008-don't-panic.md)
 - [RFC-009: Consensus Parameter Upgrades](./rfc-009-consensus-parameter-upgrades.md)
 - [RFC-010: P2P Light Client](./rfc-010-p2p-light-client.rst)
+- [RFC-011: Block Retention](./rfc-011-block-retention.md)
```

Comment:
Do we want these to be RFCs? It seems like spec RFCs are used more like what we call ADRs in the core.

Reply:
Good question. I don't think all the RFC's in the spec can be mapped to ADR's in Core (I think the ABCI++ RFC for example was more RFCy than others) but I think there are a few where you're right. I thought, however, it might be easier for people to find if we kept them in the RFC section of Tendermint.

Reply:
Yeah—I was mainly reacting to the format of the RFC docs we're importing, which uses the same (or very similar) structure with "decision" and "pro/con/neutral" sections.

I suppose we could also keep them in `spec/rfc` if we aren't sure.